main.go

```diff
@@ -48,6 +49,11 @@ var (
 	metricsAddr          string
 	enableLeaderElection bool
 	probeAddr            string
+	// WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
+	// which specifies the Namespace to watch.
+	// An empty value means the operator is running with cluster scope.
+	watchNamespaceEnvVar = "WATCH_NAMESPACE"
+	logModeEnvVar        = "LOG_MODE"
```

Comment:
Maybe give a hint what this is good for and what values the user can set.
Is the name LOG_MODE maybe misleading for a variable that can only be `true` or `false`? What if someone puts "DEBUG" or "WARN" in there?